apps/src/music/progress/MusicValidator.ts

```diff
@@ -414,4 +433,33 @@ export default class MusicValidator extends Validator {
   getValidationResults(): ValidationResult[] | undefined {
     return undefined;
   }
+
```

Comment:
I recall there is a potential extension to this functionality which would allow some events in the exemplar to be optional.  If it comes to that, and we need an algorithm to verify that a sequence exists without being at specific measures, then feel free to borrow the code from [here](https://github.com/code-dot-org/code-dot-org/pull/63957/files#diff-e1ba3cf2dc09ee0293a806e13448f2f2dfe367a8b7ecad9d55573051fb646407).